db/scripts/sanitise.sql

```diff
@@ -0,0 +1,37 @@
+TRUNCATE TABLE audit_data;
+TRUNCATE TABLE emergency_login_keys;
+TRUNCATE TABLE sessions;
+
+UPDATE account_feedbacks
+       SET suggestions=NULL;
+
+UPDATE general_feedbacks
+       SET comment=NULL,
+           email=concat('anonymised-feedback-',id,'@example.org')
+
+UPDATE jobseekers
+       SET email=concat('anonymised-jobseeker-',id,'@example.org'),
+           encrypted_password='ABCDEFGH12345',
+           reset_password_token='12345ABCDEFGH',
+           current_sign_in_ip='8.8.8.8',
+           last_sign_in_ip='8.8.4.4',
+           confirmation_token='CDEFGHIJ34567',
+           unconfirmed_email=concat('anonymised-jobseeker-',id,'@example.org'),
+           unlock_token='34567CDEFGHIJ'
+       WHERE email NOT LIKE '%@digital.education.gov.uk'
+             AND email NOT LIKE '%@education.gov.uk';
```

Comment:
Could be one line? `WHERE email NOT LIKE '%education.gov.uk';`

Reply:
Yes, that's a helpful improvement. Thanks.